setup.cfg

```
@@ -57,7 +57,7 @@ numba_extensions =
     init = vector:register_numba
 
 [flake8]
-ignore = E203, E231, E501, E722, W503, B950
-select = C,E,F,W,T,B,B9
+extend-ignore = E203, E231, E501, E722, W503, B950, D103, D102, D101, D100, D107, D105, D205, D400, D401, D104, D412
+extend-select = C,E,F,W,T,B,B9
```

```suggestion
extend-select = B9
```

The extend doesn't need the "default" values. 85% sure that includes T and B.